pom.xml

```diff
@@ -230,6 +334,20 @@
                         <arg>java.base/sun.util.logging=openjceplus</arg>
                         <arg>--add-exports </arg>
                         <arg>java.base/jdk.internal.logger=openjceplus</arg>
+                        <!--
+                        Exports statements below this line are necessary for
+                        compiling only the test source with target `test-compile`.
+                        -->
+                        <arg>--add-exports </arg>
```

Comment:
Since those are exactly the same as above, with only the part after `=` changing from `openjceplus` to `ALL-UNNAMED`, could that be a variable set as part of the profile?

Reply:
I moved what I could to the common location and exported to a target location based upon active profile. 

I did however see that a few of these exports are only needed to run tests against the bundled build. These are not there currently. I also enhanced the comment accordingly.